app/main.py

```diff
@@ -1,3 +1,5 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
+    converted_phrase = phrase.lower()
+    counter = converted_phrase.count(letter.lower())
+    return counter
```

Comment:
The `pass` statement is unnecessary here since the function already returns a value. You can safely remove this line.